Description: Corrects some invalid paths to LCL components
and removes dependency on TurboPowerIPro
Forwarded: yes
Author: Matthias Klumpp <matthias@nlinux.org>
Last-Update: 2010-05-20

--- a/easymp3gain.lpi
+++ b/easymp3gain.lpi
@@ -10,18 +10,16 @@
       <TargetFileExt Value=""/>
       <Title Value="easymp3gain"/>
       <Icon Value="0"/>
-      <ActiveEditorIndexAtStart Value="0"/>
+      <ActiveWindowIndexAtStart Value="0"/>
     </General>
     <i18n>
-      <OutDir Value="../lang"/>
+      <OutDir Value="./lang"/>
     </i18n>
     <VersionInfo>
       <AutoIncrementBuild Value="True"/>
-      <CurrentMajorRevNr Value="4"/>
-      <CurrentMinorRevNr Value="4"/>
-      <CurrentBuildNr Value="125"/>
-      <ProjectVersion Value="0.4.4.0"/>
-      <LegalCopyright Value="(c) by Giantics"/>
+      <MinorVersionNr Value="5"/>
+      <BuildNr Value="125"/>
+      <StringTable Comments="" CompanyName="" FileDescription="" FileVersion="0.5.0.125" InternalName="" LegalCopyright="(c) by Giantics" LegalTrademarks="" OriginalFilename="" ProductName="" ProductVersion="0.5.0"/>
     </VersionInfo>
     <PublishOptions>
       <Version Value="2"/>
@@ -35,31 +33,25 @@
         <LaunchingApplication PathPlusParams="/usr/X11R6/bin/xterm -T 'Lazarus Run Output' -e $(LazarusDir)/tools/runwait.sh $(TargetCmdLine)"/>
       </local>
     </RunParams>
-    <RequiredPackages Count="4">
+    <RequiredPackages Count="3">
       <Item1>
-        <PackageName Value="TurboPowerIPro"/>
+        <PackageName Value="SynEdit"/>
         <MinVersion Major="1" Valid="True"/>
       </Item1>
       <Item2>
-        <PackageName Value="SynEdit"/>
-        <MinVersion Major="1" Valid="True"/>
+        <PackageName Value="LCL"/>
       </Item2>
       <Item3>
-        <PackageName Value="LCL"/>
-      </Item3>
-      <Item4>
         <PackageName Value="FCL"/>
         <MinVersion Major="1" Valid="True"/>
-      </Item4>
+      </Item3>
     </RequiredPackages>
-    <Units Count="43">
+    <Units Count="11">
       <Unit0>
         <Filename Value="easymp3gain.lpr"/>
         <IsPartOfProject Value="True"/>
         <UnitName Value="easymp3gain"/>
-        <CursorPos X="29" Y="4"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="200"/>
+        <UsageCount Value="20"/>
       </Unit0>
       <Unit1>
         <Filename Value="unitmain.pas"/>
@@ -67,419 +59,94 @@
         <ComponentName Value="frmMp3GainMain"/>
         <ResourceBaseClass Value="Form"/>
         <UnitName Value="UnitMain"/>
-        <CursorPos X="30" Y="393"/>
-        <TopLine Value="355"/>
+        <IsVisibleTab Value="True"/>
         <EditorIndex Value="0"/>
-        <UsageCount Value="200"/>
-        <Bookmarks Count="2">
-          <Item0 X="24" Y="420" ID="0"/>
-          <Item1 X="1" Y="522" ID="1"/>
-        </Bookmarks>
+        <WindowIndex Value="0"/>
+        <TopLine Value="236"/>
+        <CursorPos X="17" Y="241"/>
+        <UsageCount Value="20"/>
         <Loaded Value="True"/>
       </Unit1>
       <Unit2>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/include/customlistview.inc"/>
-        <CursorPos X="21" Y="260"/>
-        <TopLine Value="240"/>
-        <UsageCount Value="6"/>
-      </Unit2>
-      <Unit3>
-        <Filename Value="../Mp3GainGUI/unitmain.lfm"/>
-        <CursorPos X="1" Y="1"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="6"/>
-        <SyntaxHighlighter Value="LFM"/>
-      </Unit3>
-      <Unit4>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/include/listitem.inc"/>
-        <CursorPos X="1" Y="314"/>
-        <TopLine Value="299"/>
-        <UsageCount Value="6"/>
-      </Unit4>
-      <Unit5>
         <Filename Value="unitinfo.pas"/>
         <IsPartOfProject Value="True"/>
         <ComponentName Value="frmMP3GainGUIInfo"/>
         <ResourceBaseClass Value="Form"/>
         <UnitName Value="UnitInfo"/>
-        <CursorPos X="29" Y="4"/>
-        <TopLine Value="4"/>
-        <UsageCount Value="200"/>
-      </Unit5>
-      <Unit6>
+        <UsageCount Value="20"/>
+      </Unit2>
+      <Unit3>
         <Filename Value="unitmp3gain.pas"/>
         <IsPartOfProject Value="True"/>
         <UnitName Value="UnitMP3Gain"/>
-        <CursorPos X="29" Y="4"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="201"/>
-      </Unit6>
-      <Unit7>
-        <Filename Value="../Mp3GainGUI/unitthread.pas"/>
-        <UnitName Value="UnitThread"/>
-        <CursorPos X="1" Y="17"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="16"/>
-      </Unit7>
-      <Unit8>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/include/control.inc"/>
-        <CursorPos X="1" Y="1596"/>
-        <TopLine Value="1581"/>
-        <UsageCount Value="2"/>
-      </Unit8>
-      <Unit9>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/include/menuitem.inc"/>
-        <CursorPos X="1" Y="263"/>
-        <TopLine Value="248"/>
-        <UsageCount Value="1"/>
-      </Unit9>
-      <Unit10>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/interfaces/gtk/gtkproc.inc"/>
-        <CursorPos X="1" Y="3739"/>
-        <TopLine Value="3724"/>
-        <UsageCount Value="2"/>
-      </Unit10>
-      <Unit11>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/comctrls.pp"/>
-        <UnitName Value="ComCtrls"/>
-        <CursorPos X="29" Y="16"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="1"/>
-      </Unit11>
-      <Unit12>
+        <UsageCount Value="20"/>
+      </Unit3>
+      <Unit4>
         <Filename Value="unitgainconstant.pas"/>
         <IsPartOfProject Value="True"/>
         <ComponentName Value="frmMP3GainConstant"/>
         <ResourceBaseClass Value="Form"/>
         <UnitName Value="UnitGainConstant"/>
-        <CursorPos X="29" Y="4"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="200"/>
-      </Unit12>
-      <Unit13>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/controls.pp"/>
-        <UnitName Value="Controls"/>
-        <CursorPos X="14" Y="1083"/>
-        <TopLine Value="1048"/>
-        <UsageCount Value="6"/>
-      </Unit13>
-      <Unit14>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/lcl/include/listitem.inc"/>
-        <CursorPos X="1" Y="552"/>
-        <TopLine Value="531"/>
-        <UsageCount Value="7"/>
-      </Unit14>
-      <Unit15>
-        <Filename Value="../../../../../../../../usr/share/fpcsrc/rtl/linux/ostypes.inc"/>
-        <CursorPos X="3" Y="89"/>
-        <TopLine Value="74"/>
-        <UsageCount Value="5"/>
-      </Unit15>
-      <Unit16>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/include/toolbutton.inc"/>
-        <CursorPos X="1" Y="122"/>
-        <TopLine Value="107"/>
-        <UsageCount Value="2"/>
-      </Unit16>
-      <Unit17>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/include/wincontrol.inc"/>
-        <CursorPos X="1" Y="4652"/>
-        <TopLine Value="4637"/>
-        <UsageCount Value="2"/>
-      </Unit17>
-      <Unit18>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/interfaces/gtk/gtkcallback.inc"/>
-        <CursorPos X="1" Y="1679"/>
-        <TopLine Value="1664"/>
-        <UsageCount Value="2"/>
-      </Unit18>
-      <Unit19>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/include/progressbar.inc"/>
-        <CursorPos X="1" Y="163"/>
-        <TopLine Value="148"/>
-        <UsageCount Value="1"/>
-      </Unit19>
-      <Unit20>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/interfaces/gtk/gtkwscomctrls.pp"/>
-        <UnitName Value="GtkWSComCtrls"/>
-        <CursorPos X="27" Y="278"/>
-        <TopLine Value="263"/>
-        <UsageCount Value="1"/>
-      </Unit20>
-      <Unit21>
+        <UsageCount Value="20"/>
+      </Unit4>
+      <Unit5>
         <Filename Value="unitoptions.pas"/>
         <IsPartOfProject Value="True"/>
         <ComponentName Value="frmMp3GainOptions"/>
         <ResourceBaseClass Value="Form"/>
         <UnitName Value="UnitOptions"/>
-        <CursorPos X="29" Y="4"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="201"/>
-      </Unit21>
-      <Unit22>
-        <Filename Value="../../../../../../../../usr/share/lazarus/lcl/include/imglist.inc"/>
-        <CursorPos X="1" Y="356"/>
-        <TopLine Value="341"/>
-        <UsageCount Value="1"/>
-      </Unit22>
-      <Unit23>
+        <UsageCount Value="20"/>
+      </Unit5>
+      <Unit6>
         <Filename Value="unitconsoleoutput.pas"/>
         <IsPartOfProject Value="True"/>
         <ComponentName Value="frmMP3GainConsoleOutput"/>
         <ResourceBaseClass Value="Form"/>
         <UnitName Value="unitConsoleOutput"/>
-        <CursorPos X="29" Y="4"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="200"/>
-      </Unit23>
-      <Unit24>
-        <Filename Value="callbackprocess.pas"/>
-        <UnitName Value="callbackprocess"/>
-        <CursorPos X="35" Y="72"/>
-        <TopLine Value="71"/>
-        <UsageCount Value="10"/>
-      </Unit24>
-      <Unit25>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/lcl/translations.pas"/>
-        <UnitName Value="Translations"/>
-        <CursorPos X="1" Y="477"/>
-        <TopLine Value="456"/>
-        <UsageCount Value="7"/>
-      </Unit25>
-      <Unit26>
+        <UsageCount Value="20"/>
+      </Unit6>
+      <Unit7>
         <Filename Value="unittranslate.pas"/>
         <IsPartOfProject Value="True"/>
         <UnitName Value="unitTranslate"/>
-        <CursorPos X="29" Y="4"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="201"/>
-      </Unit26>
-      <Unit27>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/lcl/include/wincontrol.inc"/>
-        <CursorPos X="1" Y="4939"/>
-        <TopLine Value="4918"/>
-        <UsageCount Value="6"/>
-      </Unit27>
-      <Unit28>
+        <UsageCount Value="20"/>
+      </Unit7>
+      <Unit8>
         <Filename Value="unitmediagain.pas"/>
         <IsPartOfProject Value="True"/>
         <UnitName Value="UnitMediaGain"/>
-        <CursorPos X="29" Y="4"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="200"/>
-      </Unit28>
-      <Unit29>
+        <UsageCount Value="20"/>
+      </Unit8>
+      <Unit9>
         <Filename Value="unitvorbisgain.pas"/>
         <IsPartOfProject Value="True"/>
         <UnitName Value="UnitVorbisGain"/>
-        <CursorPos X="29" Y="4"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="202"/>
-      </Unit29>
-      <Unit30>
+        <UsageCount Value="20"/>
+      </Unit9>
+      <Unit10>
         <Filename Value="unitvorbiscomment.pas"/>
         <IsPartOfProject Value="True"/>
         <UnitName Value="VorbisComment"/>
-        <CursorPos X="6" Y="1"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="178"/>
-      </Unit30>
-      <Unit31>
-        <Filename Value="vorbiscomment.pas"/>
-        <UnitName Value="VorbisComment"/>
-        <CursorPos X="1" Y="1"/>
-        <TopLine Value="42"/>
-        <UsageCount Value="18"/>
-      </Unit31>
-      <Unit32>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/lcl/include/control.inc"/>
-        <CursorPos X="1" Y="2095"/>
-        <TopLine Value="2074"/>
-        <UsageCount Value="7"/>
-      </Unit32>
-      <Unit33>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/lcl/include/toolbutton.inc"/>
-        <CursorPos X="1" Y="140"/>
-        <TopLine Value="120"/>
-        <UsageCount Value="3"/>
-      </Unit33>
-      <Unit34>
-        <Filename Value="../../../../../../../../usr/share/fpcsrc/2.2.0/rtl/objpas/sysutils/sysutilh.inc"/>
-        <CursorPos X="4" Y="36"/>
-        <TopLine Value="15"/>
-        <UsageCount Value="6"/>
-      </Unit34>
-      <Unit35>
-        <Filename Value="unithelp.pas"/>
-        <ComponentName Value="frmMP3GainHelp"/>
-        <ResourceBaseClass Value="Form"/>
-        <UnitName Value="unitHelp"/>
-        <CursorPos X="34" Y="44"/>
-        <TopLine Value="8"/>
-        <UsageCount Value="22"/>
-      </Unit35>
-      <Unit36>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/components/turbopower_ipro/iphtml.pas"/>
-        <UnitName Value="IpHtml"/>
-        <CursorPos X="1" Y="18139"/>
-        <TopLine Value="18117"/>
-        <UsageCount Value="7"/>
-      </Unit36>
-      <Unit37>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/lcl/helpintfs.pas"/>
-        <UnitName Value="HelpIntfs"/>
-        <CursorPos X="1" Y="331"/>
-        <TopLine Value="311"/>
-        <UsageCount Value="7"/>
-      </Unit37>
-      <Unit38>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/0.9.28/lcl/translations.pas"/>
-        <UnitName Value="Translations"/>
-        <CursorPos X="1" Y="1"/>
-        <TopLine Value="34"/>
-        <UsageCount Value="8"/>
-      </Unit38>
-      <Unit39>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/0.9.28.2/examples/hello.pp"/>
-        <UnitName Value="Hello_world"/>
-        <CursorPos X="1" Y="31"/>
-        <TopLine Value="1"/>
-        <UsageCount Value="8"/>
-      </Unit39>
-      <Unit40>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/0.9.28.2/lcl/translations.pas"/>
-        <UnitName Value="Translations"/>
-        <CursorPos X="56" Y="675"/>
-        <TopLine Value="668"/>
-        <UsageCount Value="10"/>
-      </Unit40>
-      <Unit41>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/0.9.28.2/lcl/include/control.inc"/>
-        <CursorPos X="1" Y="2644"/>
-        <TopLine Value="2625"/>
-        <UsageCount Value="8"/>
-      </Unit41>
-      <Unit42>
-        <Filename Value="../../../../../../../../usr/lib/lazarus/0.9.28.2/lcl/lclproc.pas"/>
-        <UnitName Value="LCLProc"/>
-        <CursorPos X="1" Y="1505"/>
-        <TopLine Value="1486"/>
-        <UsageCount Value="10"/>
-      </Unit42>
+        <UsageCount Value="20"/>
+      </Unit10>
     </Units>
-    <JumpHistory Count="28" HistoryIndex="27">
+    <JumpHistory Count="4" HistoryIndex="3">
       <Position1>
         <Filename Value="unitmain.pas"/>
-        <Caret Line="423" Column="1" TopLine="404"/>
+        <Caret Line="1" Column="1" TopLine="1"/>
       </Position1>
       <Position2>
         <Filename Value="unitmain.pas"/>
-        <Caret Line="425" Column="1" TopLine="406"/>
+        <Caret Line="679" Column="9" TopLine="655"/>
       </Position2>
       <Position3>
         <Filename Value="unitmain.pas"/>
-        <Caret Line="433" Column="1" TopLine="414"/>
+        <Caret Line="168" Column="24" TopLine="146"/>
       </Position3>
       <Position4>
         <Filename Value="unitmain.pas"/>
-        <Caret Line="435" Column="1" TopLine="416"/>
+        <Caret Line="651" Column="14" TopLine="632"/>
       </Position4>
-      <Position5>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="558" Column="40" TopLine="532"/>
-      </Position5>
-      <Position6>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="551" Column="32" TopLine="532"/>
-      </Position6>
-      <Position7>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="552" Column="1" TopLine="533"/>
-      </Position7>
-      <Position8>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="553" Column="1" TopLine="534"/>
-      </Position8>
-      <Position9>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="554" Column="30" TopLine="544"/>
-      </Position9>
-      <Position10>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="771" Column="3" TopLine="764"/>
-      </Position10>
-      <Position11>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="770" Column="74" TopLine="764"/>
-      </Position11>
-      <Position12>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="763" Column="33" TopLine="751"/>
-      </Position12>
-      <Position13>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="959" Column="3" TopLine="954"/>
-      </Position13>
-      <Position14>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="386" Column="46" TopLine="368"/>
-      </Position14>
-      <Position15>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="727" Column="47" TopLine="722"/>
-      </Position15>
-      <Position16>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="728" Column="27" TopLine="722"/>
-      </Position16>
-      <Position17>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="143" Column="34" TopLine="128"/>
-      </Position17>
-      <Position18>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="229" Column="10" TopLine="211"/>
-      </Position18>
-      <Position19>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="354" Column="3" TopLine="334"/>
-      </Position19>
-      <Position20>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="8" Column="40" TopLine="1"/>
-      </Position20>
-      <Position21>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="160" Column="27" TopLine="142"/>
-      </Position21>
-      <Position22>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="518" Column="17" TopLine="489"/>
-      </Position22>
-      <Position23>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="554" Column="39" TopLine="536"/>
-      </Position23>
-      <Position24>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="36" Column="50" TopLine="30"/>
-      </Position24>
-      <Position25>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="160" Column="27" TopLine="142"/>
-      </Position25>
-      <Position26>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="518" Column="17" TopLine="500"/>
-      </Position26>
-      <Position27>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="564" Column="39" TopLine="537"/>
-      </Position27>
-      <Position28>
-        <Filename Value="unitmain.pas"/>
-        <Caret Line="187" Column="30" TopLine="171"/>
-      </Position28>
     </JumpHistory>
   </ProjectOptions>
   <CompilerOptions>
@@ -507,70 +174,4 @@
       <CompilerPath Value="$(CompPath)"/>
     </Other>
   </CompilerOptions>
-  <Debugging>
-    <BreakPoints Count="3">
-      <Item1>
-        <Source Value="../MP3-Manager/main.pas"/>
-        <Line Value="289"/>
-      </Item1>
-      <Item2>
-        <Source Value="../lt/unit1.pas"/>
-        <Line Value="43"/>
-      </Item2>
-      <Item3>
-        <Source Value="../lt/unit1.pas"/>
-        <Line Value="53"/>
-      </Item3>
-    </BreakPoints>
-    <Watches Count="10">
-      <Item1>
-        <Expression Value="PathDelim"/>
-        <Enabled Value="False"/>
-      </Item1>
-      <Item2>
-        <Expression Value="Text"/>
-        <Enabled Value="False"/>
-      </Item2>
-      <Item3>
-        <Expression Value="SongItem"/>
-        <Enabled Value="False"/>
-      </Item3>
-      <Item4>
-        <Expression Value="(SongItem^.ListViewItem)"/>
-        <Enabled Value="False"/>
-      </Item4>
-      <Item5>
-        <Expression Value="pmnAnalysisTrack"/>
-        <Enabled Value="False"/>
-      </Item5>
-      <Item6>
-        <Expression Value="SongItems"/>
-        <Enabled Value="False"/>
-      </Item6>
-      <Item7>
-        <Expression Value="Location"/>
-        <Enabled Value="False"/>
-      </Item7>
-      <Item8>
-        <Expression Value="Testitem"/>
-        <Enabled Value="False"/>
-      </Item8>
-      <Item9>
-        <Expression Value="TFPList(TMediaGainTaskList(TaskList).FList).FList"/>
-        <Enabled Value="False"/>
-      </Item9>
-      <Item10>
-        <Expression Value="TFPList(TMediaGainTaskList(TaskList).FList)"/>
-        <Enabled Value="False"/>
-      </Item10>
-    </Watches>
-    <Exceptions Count="2">
-      <Item1>
-        <Name Value="ECodetoolError"/>
-      </Item1>
-      <Item2>
-        <Name Value="EFOpenError"/>
-      </Item2>
-    </Exceptions>
-  </Debugging>
 </CONFIG>
